integration/Taskfile.yml

```diff
@@ -3,9 +3,10 @@
 version: 3
 
 vars:
+  COUNT: 10
   BENCHTIME: 5s
-  BENCHNAME: '{{default "." .BENCHNAME}}'
-  RACEFLAG: -race={{ne OS "windows"}}
+  BENCHNAME: .
```

Comment:
Why are we removing option to provide benchmark name? It's useful for benchmark development.

Reply:
That's the default value, it could be overwritten. The old value did essentially the same, but in a more complicated way

Reply:
I think `task target VAR=value` should work

Reply:
It works, thanks!